client/src/Components/Box/RankingBox.tsx

```diff
@@ -11,14 +11,16 @@ export const RankingBox: FC<BookDetails> = ({ title, id, rating, author, review,
     history.push(path);
   };
 
+  const score = rating === 'NaN' ? 'brak ocen' : rating;
```

Comment:
Powinno zadziałać ale zdecydowanie lepszym rozwiązaniem byłoby użycie motody inNaN np ;) 
Zamieńmy to proszę